watertap/examples/flowsheets/oaro/oaro_multi.py

```diff
@@ -0,0 +1,797 @@
+
+    if erd_type == ERDtype.pump_as_turbine:
+        for erd in m.fs.EnergyRecoveryDevices.values():
+            iscale.set_scaling_factor(erd.control_volume.work, 1e-3)
+    else:
+        erd_type_not_found(erd_type)
+    # unused scaling factors needed by IDAES base costing module
+    # calculate and propagate scaling factors
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def set_operating_conditions(
+    m,
+    solver=None,
+):
+
+    if solver is None:
+        solver = get_solver()
+
+    # ---specifications---
+    # feed
+    # state variables
+    pressure_atmospheric = 101325
+    feed_temperature = 273.15 + 25
+    m.fs.feed.properties[0].pressure.fix(pressure_atmospheric)  # feed pressure [Pa]
+    m.fs.feed.properties[0].temperature.fix(feed_temperature)  # feed temperature [K]
+
+    # properties (cannot be fixed for initialization routines, must calculate the state variables)
+    feed_flow_mass = 1
+    feed_mass_frac_NaCl = 0.07
+    feed_mass_frac_H2O = 1 - feed_mass_frac_NaCl
+    m.fs.feed.properties[0].flow_mass_phase_comp["Liq", "NaCl"].fix(
+        feed_flow_mass * feed_mass_frac_NaCl
+    )
+    m.fs.feed.properties[0].flow_mass_phase_comp["Liq", "H2O"].fix(
+        feed_flow_mass * feed_mass_frac_H2O
+    )
+
+    # primary pumps
+    for pump in m.fs.PrimaryPumps.values():
+        pump.control_volume.properties_out[0].pressure = 50e5
+        pump.efficiency_pump.fix(0.80)
+        pump.control_volume.properties_out[0].pressure.fix()
+
+    for pump in m.fs.RecyclePumps.values():
+        pump.control_volume.properties_out[0].pressure = 4e5
+        pump.efficiency_pump.fix(0.80)
+        pump.control_volume.properties_out[0].pressure.fix()
+
+    # initial guess for states of recycle pumps (temperature and concentrations)
+    for stage in m.fs.NonFirstStages:
+        m.fs.RecyclePumps[stage].control_volume.properties_out[
+            0
+        ].temperature.value = feed_temperature
+        permeate_flow_mass = 0.75
+        permeate_mass_frac_NaCl = 0.02
+        permeate_mass_frac_H2O = 1 - permeate_mass_frac_NaCl
+        m.fs.RecyclePumps[stage].control_volume.properties_out[0].flow_mass_phase_comp[
+            "Liq", "H2O"
+        ].value = (permeate_flow_mass * permeate_mass_frac_H2O)
+        m.fs.RecyclePumps[stage].control_volume.properties_out[0].flow_mass_phase_comp[
+            "Liq", "NaCl"
+        ].value = (permeate_flow_mass * permeate_mass_frac_NaCl)
+
+    # Initialize OARO
+    membrane_area = 50
+    A = 4.2e-12
+    B = 1.3e-8
+
+    for stage in m.fs.OAROUnits.values():
+
+        stage.area.fix(membrane_area)
+
+        stage.A_comp.fix(A)
+        stage.B_comp.fix(B)
+
+        stage.structural_parameter.fix(300e-6)
+
+        stage.permeate_side.channel_height.fix(0.001)
+        stage.permeate_side.spacer_porosity.fix(0.75)
+        stage.feed_side.channel_height.fix(0.002)
+        stage.feed_side.spacer_porosity.fix(0.75)
```

Comment:
In reality the feed and permeate side spacer porosities would be different, but I think we could use the same here and consider increasing to 0.85 (value we chose in LSRRO paper)